phalcon/cache/backend/xcache.zep

```diff
@@ -195,7 +195,7 @@ class Xcache extends Backend
 	 * @param int|string keyName
 	 * @return boolean
 	 */
-	public function delete(var keyName)
+	public function delete(var keyName) -> boolean
```

Comment:
This class should be removed from 4.0.x branch at all (in the separated PR of course).

/cc @niden

Reply:
We can just leave this one in there for now. I will create a different PR to remove xcache, memcache and apc from v4 since they are all deprecated